R/dataset.R

```diff
@@ -137,7 +137,24 @@ Dataset.data <- function(id, filters, row.names = NULL, env = solvebio:::.solveE
 
     tryCatch({
         res <- .request('POST', path=path, body=body, env=env)
-        return(formatSolveBioQueryResponse(res, row.names=row.names))
+        res <- formatSolveBioQueryResponse(res)
+        if (!is.null(col.names)) {
+            # Create a dataframe that maps column names with titles
+            # Append column "_id" to the list of names and titles because it's always present in result query, but not in names and titles
```

Comment:
`_id` may not always be present, for example if I run a query like this: `Dataset.query(id, fields=list("other", "fields"))`


Tried this and got:

```
> query <- Dataset.query(dataset$id, fields=list("fmi_sample_id"))
Query failed: attempt to set 'colnames' on an object with less than two dimensions
```

Reply:
You can probably just check the `res` to see if _id is in there though.